src/config/packages/dev/debug.yaml

```diff
@@ -0,0 +1,4 @@
+debug:
+    # Forwards VarDumper Data clones to a centralized server allowing to inspect dumps on CLI or in your browser.
+    # See the "server:dump" command to start a new server.
+    dump_destination: "tcp://%env(VAR_DUMPER_SERVER)%"
```

Comment:
Is this setting correct? Did you add this to the docker-compose.dev.yml? :thinking:

Reply:
I remember adding symfony/var-dumper to inspect something.